### DISCUSSION TITLE
PorousFlow - Simulation diverged

### DISCUSSION THREAD

**Discussion by josebastiase** [idaholab/moose#21708] [Q&A Modules: General] [answered]
Hi all,
I got a hydro-Mechanical simulation on PorousFlow that diverges even though the residuals are decreasing. As shown here

Time Step 2, time = 2.1, dt = 1.1
 0 Nonlinear |R| = **9.898234e+06**
      0 Linear |R| = 9.898234e+06
      1 Linear |R| = 7.617497e-09
 1 Nonlinear |R| = **7.288478e+00**
      0 Linear |R| = 7.288478e+00
      1 Linear |R| = 5.361864e-11
 2 Nonlinear |R| = **3.644174e+00**
      0 Linear |R| = 3.644174e+00
      1 Linear |R| = 1.679125e-11
Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH iterations 2
 Solve Did NOT Converge!
Aborting as solve did not converge

I am coupling PorousFlow variables to several ODE Kernels. The poor convergence comes from there. 
My preconditioner 

[Preconditioning]
  [lu]
    type = SMP
    full = true
    petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
    petsc_options_value = ' lu       mumps'
  []
[]

thanks in advance!
Jose

**Comment by maxnezdyur** [comment, marked as answer]
Not sure if this will help. But in the executioner block try line_search = 'none' or line_search = 'bt' . I sometimes have problems with the default line_search option.

**Reply by josebastiase** [reply]
Thanks! line_search = 'none' worked. Also scaling reduce the initial residuals. 
Cheers!
Jose
